src/clj/game/cards/events.clj

```diff
@@ -1051,6 +1051,19 @@
     :choices (req runnable-servers)
     :effect (effect (run target nil card))}
 
+   "Insight"
+   {:async true
+    :implementation "Rearranging R&D is not optional"
```

Comment:
What does this mean? Is the Corp not allowed to re-arrange cards into the same arrangement?

Reply:
What I mean is that the card says the Corp 'may' rearrange, but I think asking the corp whether they want to will only cause confusion, so I assume the Corp wants to rearrange and rely on them to keep the order as-is if they don't want to. Maybe this note should just be deleted - seems like it causes confusion too. :)